src/concept-coach/index.cjsx

```diff
@@ -63,12 +78,23 @@ class ConceptCoachAPI extends EventEmitter2
 
     _.defaults(navOptions, {prefix: '/', base: 'concept-coach/'})
 
-    restAPI.initialize(baseUrl)
-    navigation.initialize(navOptions)
+    restAPI.init = _.partial restAPI.initialize, baseUrl
+    navigation.init = _.partial navigation.initialize, navOptions
+
+    @models = [restAPI, navigation, User, exercise, progress, task, componentModel]
+    initializeModels(@models)
 
     listenAndBroadcast(@)
     setupAPIListeners(@)
-    User.ensureStatusLoaded()
+    User.ensureStatusLoaded(true)
+
+  destroy: ->
```

Comment:
Does this `destroy` call all the others? (I'm assuming this is the one webview will end up calling)